Vegeting/Screens/RegisterUserProfile/Component/SelectVegetarianTypeViewController.swift

```diff
@@ -0,0 +1,150 @@
+//
+//  SelectVegetarianTypeViewController.swift
+//  Vegeting
+//
+//  Created by 김원희 on 2022/11/17.
+//
+
+import UIKit
+
+struct TypeDescription {
+    let type: String
+    let description: String
+}
```

Comment:
p3;
enum으로 관리하는 건 어떨까요? 해당 내용이 다른 View에서도 쓰일거 같고, enum을 통해서 가독성이나 재사용성을 높이면 더욱 좋을거 같습니다!
```suggestion
enum vegetarianType {
    case vegan
    // ...
    var description: String { 
       switch self {
       case .vegan:
       return "완전한 식물성만 섭취해요"
       }
    }
}
```

Reply:
좋은 의견 감사합니다!!
[반영](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Vegeting/pull/146/files#diff-cff65fe87ac40561d8db62b597edd81f8fa0b5c961fdd3084d1e2db824ec6c56R10-R44)해보았습니다~~